<p>This rule does nothing. Absolutely nothing. Apart from wasting your time. So a bit more than nothing...</p>

<h2>Noncompliant Code Example</h2>
<pre>
  MESSAGE 'NoOp'. // This is valid
</pre>

<h2>Compliant Solution</h2>
<pre>
  MESSAGE 'NoOp'. // This is still valid
</pre>
